### ISSUE TITLE
Feedback - General Navigation

### ISSUE THREAD

**Issue by freeranger** [AgileVentures/av-static-website#36]
I browse to http://showcase.agileventures.org.

I like the colour scheme - it feels like an evolution of the main AV scheme - more modern and not as harsh on the eyeballs.  I'd like to see this across the entire site.

The front page doesn't tell me much to be honest - I have to move on to find anything useful, so maybe that initial page doesn't serve any purpose?   It's just some text and a button...

This is the showcase site for AV - so why isn't the showcase front and centre?  I have an initial page that tells me nothing really, it "wants" me to click on the big "Tell Me More" button which leads on to a page about why I should choose AV - again, no showcase!

Anyway, I click on the button and I like the vertical carousel effect to bring the content into view.
This content is better on the main site - I've already been on agileventures.org and read about who you are and what you are doing, so I am coming here to see examples of your work - I don't really need to be told who you are and what you do again. 

Still looking for that showcase but now I am confused.
I clicked on "Tell Me More", so in my mind I am in "button clicking mode" so I expect there to be another button to click or other such navigation in the page, but there isn't.

I can see that the menu has "Why Us" highlighted - is this the same place I navigated to?  It isn't what I clicked...
It is a bit jarring to move from "in page" navigation to using the menu system - I'd rather use one or the other, and have the highlighted menu item match whatever the button was I clicked if indeed we need both buttons and menu items.

I want to go back to the front page incase I missed a button there so I click the "back" button in the browser - oops, I am taken to the previous site I was on, not the previous page I was on in this site - not a great user experience.  It is possible to make SPA's work with the back button - as a user I don't know or care that it's an SPA, so I reasonably expect the back button to take me to where I was just a minute ago...

Ok now I know I need to use the menus to go to any page except "Why Us", where I have an optional big button to click.
I click on the various options and notice I get the nice carousel effect for all of them except "blog" which jars because the screen flashes a bit - this is a nav to a completely new page, not an SPA nav - and indeed I can see the URL has changed from  http://showcase.agileventures.org to http://blog.agileventures.org/#blog.  Why the double-blog?  blog. and #blog - seems a bit redundant.

I now click on the "Contact" link from the blog page and I am taken to http://showcase.agileventures.org/#contact. If I now click on any other link (except blog) this remains the same - so I click on "Why Us" and the link still says "http://showcase.agileventures.org/#contact" even though the nav actually happens.  (Incidentally, the nav from the blog page to "Why Us" leads to http://showcase.agileventures.org/#services - I'd prefer some consistency here in the name of the page, menu item and nav link).

I appreciate that "normal" users might not notice or care that the URL does not change, but you are pitching to devs to join you to work on projects as well as to non profits looking to have their visions realised - devs will notice things like URL's not changing and that you have switched from SPA to non-SPA

**Comment by freeranger** [comment]
Footnote - in resizing the window sometimes I get this:
![image](https://cloud.githubusercontent.com/assets/1401069/16081750/bb1e1086-3306-11e6-9004-6d2be180bc7d.png)

Not sure how I have "Why Us" highlighted and "Tell Me More" visible at the same time, but the odd thing was the bit of white just above the "Tell Me More" button which appeared/disappeared as I resized.

**Comment by tansaku** [comment]
I wonder if http://showcase.agileventures.org/ is the appropriate url - maybe it should be http://sales.agileventures.org/ or something else? http://charities.agileventures.org/  http://nonprofits.agileventures.org/ http://charity-solutions.agileventures.org/  http://charity-clients.agileventures.org/

**Comment by freeranger** [comment]
That depends on the purpose really as to whether or not the url is appropriate..

**Comment by tansaku** [comment]
I've switched to it: http://nonprofits.agileventures.org/

**Comment by tansaku** [comment]
So not sure if changing the name of of the site URL addresses part of this, but this site is intended to be for non-profits and to tell them about what we can do for them AND to showcase some of our work, so the flow I see is that you land on the first page, which tells people what we do:

```
Software solutions for non-profits
WEBSITES, MOBILE, APPS, PROJECTS
```

Maybe people shouldn't have to click a button to move on, but hey, then can scroll down.  Button for people who don't scroll so much.

Next section whether we click or scroll is why a nonprofit should choose AV to develop their solution.  I guess this comes from imagining that a nonprofit is considering working with us, so we want to say why we are good to work with.  We are assuming nonprofits may well not have picked up this info from the main site, even if they have seen that.

No button on "Why Agile Ventures" hmm, yes could do with a button to lead people on to the "case studies", button/nav consistency, back button - created some tickets for those (#55, #56, #57).

Fixed the blog issue (and the hashtag completions) by removing separate ghost blog, and removing blog link from navigation and switching to integrated middleman blog - blog could be more discoverable but that's a separate issue.

Might also be nice to make URL change with scroll to section - I'll open another ticket for that #58

@freeranger thanks so much for the detailed feedback on the site.  It was somewhat overwhelming to digest, but now I've been able to pull it apart into several tickets I think we are getting somewhere ...
